### ISSUE TITLE
[Bug] render thing

### ISSUE THREAD

**Issue by dadudeme** [MeteorDevelopment/meteor-client#1440]
### Describe the bug

![2021-09-01_19 27 31](https://user-images.githubusercontent.com/79927205/131724394-417eb45c-cd74-4c11-8e85-0e24b5aacb4c.png)

### Steps to reproduce

```bash
1.start minecraft with meteor 0.4.6-1234
2.open packet chanceler
3.hit select on S2C
4. exit S2C
...
```

### Meteor Version

0.4.6-1234

### Minecraft Version

1.17.1

### Operating System

{"label"=>"Windows"}

**Comment by Wide-Cat** [comment]
how tf could any of youre most recent commits fuck up the colours like that?

**Comment by MineGame159** [comment]
I cannot reproduce it with the steps you have provided

**Comment by MineGame159** [comment]
Fixed in ffa71935f100797df67267a8235ee0cd1633c10e
